pyvelox/pyvelox.h

```diff
@@ -24,6 +24,13 @@
 
 namespace facebook::velox::py {
 
+/// \defgroup pyvelox Velox Python Bindings
```

Comment:
FYI - Lets get rid of these for now. We are more interested in the python api's than the c++ python interface.